### PR TITLE
Update use-app-catalog.md

### PR DESCRIPTION
**Pull request by lucabandMSFT**
adding information for "deploy" option

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointOnline/use-app-catalog.md [hunk 1]
```diff
@@ -92,6 +92,9 @@ To get to the App Catalog site once it's been created, follow steps 1 and 2 in t
 If you want all users to use an app, you can deploy it to specific site collections, managed paths, or site templates. Deploying an app essentially installs that app on a site for users so that is available for use. Deployed apps appear on the Site Contents page for a site.
   
 You can deploy a third-party app, or you can [deploy a custom app](use-app-catalog.md#__deploy_a_custom).
+
+ > [!NOTE]
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/use-app-catalog.md]:
```suggestion
> [!NOTE]
```

File: SharePoint/SharePointOnline/use-app-catalog.md [hunk 2]
```diff
@@ -92,6 +92,9 @@ To get to the App Catalog site once it's been created, follow steps 1 and 2 in t
...
+    > This option is only available for SharePoint add-ins. To see tenant-scope deployment options for SharePoint Framework solutions see [Tenant-scoped solution deployment for SharePoint Framework solutions](https://docs.microsoft.com/en-us/sharepoint/dev/spfx/tenant-scoped-deployment).
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/use-app-catalog.md]:
```suggestion
> This option is only available for SharePoint add-ins. To see tenant-scoped deployment options for SharePoint Framework solutions, see [Tenant-scoped solution deployment for SharePoint Framework solutions](https://docs.microsoft.com/sharepoint/dev/spfx/tenant-scoped-deployment). 
```

File: SharePoint/SharePointOnline/use-app-catalog.md (hunk 1 above)
Comment (lucabandMSFT) [reply SharePoint/SharePointOnline/use-app-catalog.md]: Thanks.

**Comment by scanum** [comment]
@kaarins PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**